src/components/App.js

```diff
@@ -22,9 +19,11 @@ function App() {
             <div className="wrapper">
               <Route exact path="/" component={Splash} />
               <Route exact path="/my-profile" />
-              <Route exact path="/create-event" component={CreateEventForm}/>
-              <Route exact path="/my-events" />
-              <Route exact path="/sign-out" component={Splash} />
+              <Route exact path="/create-event" component={CreateEventForm} />
+              <Route exact path="/my-events">
+                <MyEvents events={events}/>
+              </Route>
```

Comment:
do we want to change the syntax  here to make it more consistent with splash and create event form?